Portfolio2.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <title>Elmi Farah's Portfolio'</title>
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet" type="text/css " href="portfolio2.css ">
+   </head>
+   <body>
+      <header>
+         <div id="bannertop">
+            <div class="menue">
+               <ul>
+                  <li class="logo"> <img src="maydenacademylogo.png"/> </li>
+               	  <li class="active"> ABOUT ME </li> 
+                  <li class="activ"> PORTFOLIO </li> 
+                  <li class="acti"> CONTACT INFO </li> 
```

Comment:
These don't do anything, are they meant to link to parts of the page? Inside your li elements you should probably have an a tag so you can link

Reply:
Also check the indenting, and as per my suggestion on your css file, just give these all the active class. Although i'd consider renaming the class to something else, active is usually used to mean that the user is currently on that particular link page